pinery-lims-miso/src/main/java/ca/on/oicr/pinery/lims/miso/MisoClient.java

```diff
@@ -318,8 +317,17 @@
       "        INNER JOIN Indices ON Indices.indexId = Library_Index.index_indexId\n" + 
       "                WHERE position = 2\n" + 
       "        ) bc2 ON bc2.library_libraryId = l.libraryId\n" + 
-      "LEFT JOIN BoxPosition pos ON pos.boxPositionId = l.boxPositionId\n" + 
+      "LEFT JOIN BoxPosition pos ON pos.targetId = l.libraryId\n" + 
+      "        AND pos.targetType = 'L'\n" +
```

Comment:
I know that these are `S` and `L` in the db, but @djcooke [requested](https://github.com/TGAC/miso-lims/pull/676) that @apmasell change the targetType to "something more descriptive like 'Sample' and 'SampleTissue'" and the change was made in the BoxImpl but maybe not in the db migration script. Will this cause an issue here?